content/automate/ManageIQ/AutomationManagement/AnsibleTower/Operations/Methods.class/__methods__/available_credentials.yaml

```diff
@@ -0,0 +1,32 @@
+---
+object_type: method
+version: 1.0
+object:
+  attributes:
+    name: Available_Credentials
+    display_name: 
+    description: 
+    scope: instance
+    language: ruby
+    location: inline
+  inputs:
+  - field:
+      aetype: 
+      name: credential_type
+      display_name: 
+      datatype: 
+      priority: 1
+      owner: 
+      default_value: ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential
```

Comment:
@gmcculloug 
One reason I decided to do that was to preserve the expected format of the value when the method gets copied around to a different class/domain without instances.